config/main.py

```diff
@@ -1515,13 +1516,29 @@ def replace(ctx, target_file_path, format, dry_run, ignore_non_yang_tables, igno
 
         config_format = ConfigFormat[format.upper()]
 
-        GenericUpdater().replace(target_config, config_format, verbose, dry_run, ignore_non_yang_tables, ignore_path)
+        if multi_asic.is_multi_asic():
+            scope_list = [HOST_NAMESPACE]
+            scope_list.extend(multi_asic.get_namespace_list())
+            tobevalidated = copy.deepcopy(target_config)
+            for scope in scope_list:
+                scope_config = tobevalidated.pop(scope)
+                if not SonicYangCfgDbGenerator().validate_config_db_json(scope_config):
+                    raise GenericConfigUpdaterError(f"Invalid config for {scope} in {target_file_path}")
+            config_rollbacker = MultiASICConfigRollbacker(scopelist=scope_list)
+            config_rollbacker.replace_all(target_config)
+        else:
+            if not SonicYangCfgDbGenerator().validate_config_db_json(target_config):
```

Comment:
>validate_config_db_json [](http://example.com/codeflow?start=45&length=23)

This is not called in old code for single-asic. I am wondering what is the old behavior is `target_config` is not yang validated? Will it throw some exception?